applications/DEMApplication/python_scripts/dem_default_input_parameters.py

```diff
@@ -40,6 +40,7 @@ def GetDefaultInputParameters():
             "RotationOption"                   : true,
             "CleanIndentationsOption"          : false,
             "RemoveBallsInEmbeddedOption"      : false,
+            "ComputeReactionsOnFixedParticlesOption" : false,
```

Comment:
```suggestion
            "compute_reactions_on_fixed_particles_option" : false,
```

more consistent with rest of Kratos